geography/README.md

```diff
@@ -0,0 +1,158 @@
+# Mobility Data Specification: Geography
+
+This specification contains a collection of RESTful APIs used to read Geographies (descriptions of geographical information, e.g. multi-polygons, currently represented via GeoJSON).
+
+Geographical data has many applications in the context of mobility, such as the description of municipal boundaries, locations for pick-up and drop-off zones, and areas of temporary closure for special events or emergencies.  This API is intended to support a variety of other APIs, including the Policy API.
+
+Geographical data will be stored as GeoJSON and read from either `geographies.json` or the `/geographies` endpoint, referenced by UUID. Geography data once published through this API shall be treated as immutable, to ensure that any rules or regulations referring to the boundaries cannot be retroactively changed.  A Geography may be deprecated and replaced by updated version with a new UUID.
```

Comment:
I wonder if this should be /geography and geography.json, instead of /geographies? The other APIs are singular, and match their API name, and we call this the Geography API. Also changing the `/geography/{geography_id}` endpoint.

Reply:
Consensus on call today to keep the endpoint as /geographies